libethereum/ExtVM.cpp

```diff
@@ -33,7 +33,7 @@ namespace // anonymous
 static unsigned const c_depthLimit = 1024;
 
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
-static size_t const c_singleExecutionStackSize = 6 * 1024;
+static size_t const c_singleExecutionStackSize = 9 * 1024;
```

Comment:
This was a constant which you had raised before in a previous commit, and then found a way to reduce again later, right, @chfast?    Given that ALL the tests are broken at the moment, I'll merge this PR as-is, but we might need to iterate on this again, eh?